pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -210,7 +210,7 @@ type CephBlockPoolList struct {
 
 // PoolSpec represents the spec of ceph pool
 type PoolSpec struct {
-	// The failure domain: osd/host/(region or zone if topologyAware) - technically also any type in the crush map
+	// The failure domain: osd/host/(region or zone if OSD are on PVC) - technically also any type in the crush map
```

Comment:
Aren't we doing region/zone whenever they're available?

Reply:
Right, now we do.